app/controllers/api/autocomplete_controller.rb

```diff
@@ -33,6 +33,10 @@ def create
 
     private
 
+    def has_headers?
+      @items.file_headings == %w[text value]
```

Comment:
forgot about this! 😆 

`@items.files_headings` now doesn't return things that are downcased already

Reply:
😅 good spot. Have changed this to look for the number of headings rather than the specific strings as the validator should weed out any header issues beforehand.